src/docs/docs/api.mdx

```diff
@@ -0,0 +1,102 @@
+
+Making an API public offers considerable benefits. A public API allows our customers to incorporate Sentry into their workflows. It also gives Sentry more visibility in our customers' organizations ("Woah that's cool data, where'd it come from?"..."Sentry!"). Remember, however, making an API public means that once an endpoint is public, you can only add to it: You cannot make *any* breaking changes.
+
+As a guide, use these questions:
+
+1. Is the feature for which you're making the endpoint stable?
+2. Will the API change substantially in the future?
+3. Does the API offer value to customers who are accessing it?
+
+If your answers are Yes, No and Yes, you're in business - make the API public.
+
+## How to make an endpoint public?
+
+We use [Open API Spec 3](https://swagger.io/docs/specification/about/) to write our API documentation. You can find that content here: [https://github.com/getsentry/sentry/tree/master/api-docs.](https://github.com/getsentry/sentry/tree/master/api-docs) You can find the tests here: [https://github.com/getsentry/sentry/tree/master/tests/apidocs](https://github.com/getsentry/sentry/tree/master/tests/apidocs).
+
+Once you make changes to an endpoint and merge the change into Sentry, a Github Action will update the schema in [sentry-api-schema](https://github.com/getsentry/sentry-api-schema) with a build version. To make your changes go live:
+
+1. Get the latest SHA from sentry-api-schema.
+2. Go to sentry-docs and update the SHA ([https://github.com/getsentry/sentry-docs/blob/bf5a9463df40bf6e99685f14451c6e1313898b36/src/gatsby/config.ts#L187](https://github.com/getsentry/sentry-docs/blob/bf5a9463df40bf6e99685f14451c6e1313898b36/src/gatsby/config.ts#L187)).
+
+### Requesting an API to be public
+
+Want an endpoint to be public?
+
+Look at the [issues on sentry with the `Component: API` label](https://github.com/getsentry/sentry/issues?q=is%3Aopen+is%3Aissue+label%3A%22Component%3A+API%22). If a request has already been made to make the endpoint public, give it a thumbs up. If not, create a [feature request](https://github.com/getsentry/sentry/issues/new?template=feature_request.md) on Sentry and add the `Component: API` label.
+
+The team responsible for the endpoint will review the stability of the endpoint. If the endpoint will not have breaking changes in future, they can determine whether to make it public.
+
+### FAQs
+
+**When should an attribute be `required`?**
+
+An attribute is `required` if it will always be returned by the API.
+
+**What does it mean when a response doesn't have a schema?**
+
+Some endpoints have no response schema. This means that while the endpoint is public, the attributes within that endpoint can change at any time. This is a relic from migrating the documentation from our prior approach. Note that, going forward, we recommend new endpoints always provide response schema.
+
+**I have a question and it has not been answered.**
+
+No problem. Ping the Ecosystem Team.
```

Comment:
Since we're making this public (alternate, provide an email address?)
```suggestion
No problem. Open an [issue on sentry with the `Component: API` label](https://github.com/getsentry/sentry/issues?q=is%3Aopen+is%3Aissue+label%3A%22Component%3A+API%22) so we can answer your question.
```

Reply:
Yea I like email. Don't want to open the issues floodgate for this support just yet.